x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines.tsx

```diff
@@ -65,6 +66,29 @@ export const SearchIndexPipelines: React.FC = () => {
   const { makeRequest: revertPipeline } = useActions(RevertConnectorPipelineApilogic);
   const apiIndex = isApiIndex(index);
   const extractionDisabled = getContentExtractionDisabled(index);
+  const [isRevertPipeline, setRevertPipeline] = useState(false);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+  useEffect(() => {
+    if (!isDeleteModalOpen) {
+      if (isRevertPipeline) {
+        const pipelinesButton = document.querySelector<HTMLDivElement>(
+          `[id="${SearchIndexTabId.PIPELINES}"]`
+        );
+        if (pipelinesButton) {
+          pipelinesButton.focus();
+        }
+        setRevertPipeline(false);
+      }
+    } else if (buttonRef.current) {
```

Comment:
This `else` is on the `if (!isDeleteModalOpen) {}` and it should be on the `if (isRevertPipeline) {}` instead.